### PR TITLE
March23 edits

### PR DESCRIPTION
**Pull request by OliviaRudiak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ You can check the installation by opening up a command window and typing the fol
 cd %HOMEPATH%\Projects\sentinel.example
 mvn test
 ```
-If chrome pops up and the tests run successfully, you are ready to move to Section 1.2.
+If Google Chrome pops up and the tests run successfully, you are ready to move to Section 1.2.
 
-If you have issues with installation, 
+If you have issues with installation, refer to the individual tool manufacturers. (?)
```
Comment (dougnoel) [inline README.md]: Yes, this change is correct.

File: README.md [hunk 2]
```diff
@@ -59,16 +59,16 @@ cp conf/sentinel.linux.example.yml conf/sentinel.yml
...
 
-1. On the command line change to the directory you wish to check the project out in and type `git clone https://github.com/dougnoel/sentinel.example.git`
-2. In the sentinel.example/conf directory there are a number of example configuration files. Copy the one for your operating system to a file titled `sentinel.yml`. Ensure the browser listed is one installed on your system.
-3. Change to the project `cd sentinel.example` and run `mvn test`. Provided that the configuration file exists, the project will run.
+1. On the command line/terminal, change to the directory you wish to check the project out (how do they determine which directory? Do non-technical readers know what a directory is? Is there a recommended one to use?) in and type `git clone https://github.com/dougnoel/sentinel.example.git`
+2. In the sentinel.example/conf directory there are a number of example configuration files. Copy the one for your operating system (do readers know which one aligns to their operating system?) to a file titled `sentinel.yml`. Ensure the browser listed is one installed on your system.
```
Comment (dougnoel) [inline README.md]: Readers should know their operating system, yes.

File: README.md (hunk 1 above)
Comment (dougnoel) [inline README.md]: Yes, this change is correct.

File: README.md [hunk 3]
```diff
@@ -59,16 +59,16 @@ cp conf/sentinel.linux.example.yml conf/sentinel.yml
...
 If you have issues with installation, refer to the individual tool manufacturers.
 
-## 1.2 Running the Example Tests on the Command Line
+## 1.2 Running the Example Tests on the Command Line (Terminal for Mac Users)
 If you already have all the tools installed (Git, Java, Maven), you can follow the following instructions on any operating system to get running quickly.
 
-1. On the command line change to the directory you wish to check the project out in and type `git clone https://github.com/dougnoel/sentinel.example.git`
-2. In the sentinel.example/conf directory there are a number of example configuration files. Copy the one for your operating system to a file titled `sentinel.yml`. Ensure the browser listed is one installed on your system.
-3. Change to the project `cd sentinel.example` and run `mvn test`. Provided that the configuration file exists, the project will run.
+1. On the command line/terminal, change to the directory you wish to check the project out (how do they determine which directory? Do non-technical readers know what a directory is? Is there a recommended one to use?) in and type `git clone https://github.com/dougnoel/sentinel.example.git`
```
Comment (dougnoel) [inline README.md]: Yes, the recommended directory/folder is Documents/Projects

File: README.md [hunk 4]
```diff
@@ -139,8 +139,8 @@ Feature: {StoryTicketNumber} {Subject of your Ticket}
       | BadPassword |
 ```
 
-### 2.2.2 Feature Section
-A feature is equal to an agile story. This equivalence is made to keep your feature files manageable. If your feature files are too large, this is an indication that either you are not using them correctly (multiple stories per story), your stories are too large (something to bring up with your team), or you are not pruning old tests. When adding a story that modifies an existing, story do not add in all the same tests again. Instead, use the power of tagging to tag the other features or scenarios affected with additional tags (see below about tags). If a feature is removed - remove the tests.
+### 2.2.2 Feature Section (important to distinguish the difference between Feature = Story and Feature > Story in some tools ie Rally)
```
Comment (dougnoel) [inline README.md]: Was this a note for me?

File: README.md [hunk 5]
```diff
@@ -79,34 +79,34 @@ If you already have all the tools installed (Git, Java, Maven), you can follow t
...
 
-## 2.2 Creating a Cucumber Test
-Now that we have a ticket created, we can creature our first feature file. It's important to note that we use what Cucumber calls a feature file to match what Agile typically calls a story. Depending on where you work, a story may roll up to an epic, or it may roll up to a feature, then an epic, or something else. For our purposes, a feature file contains one story and a number of test scenarios. The purpose of this is to keep our tests as small an atomic as possible. If we let our feature files cover too much, they tend to get too complex, these slow our tetsing down, and feature creep starts over running our tests.
+## 2.2 Creating a Cucumber Test (this section needs to be updated to reflect neither Jira nor Rally terminology to avoid confusion)
```
Comment (dougnoel) [inline README.md]: Other than the one comment where is this specified.

File: README.md [hunk 6]
```diff
@@ -79,34 +79,34 @@ If you already have all the tools installed (Git, Java, Maven), you can follow t
...
+Writing a test starts before any code is written. When a user story is created, a ticket (what's a ticket? assumption - a user story?) should have Acceptance Criteria that describe what the product owner expects the feature to do. Writing good Acceptance Criteria is about communicating what you expect to happen when development on a user story is complete. Good Acceptance Criteria act as your discussion points for clarifying stories, your test cases for automated testing, and your bug reports. After Acceptance criteria are written by the product owner, they should be agreed upon by the product owners, development team members, and QA *before* the ticket is accepted into a sprint. Once a ticket is in a sprint, you will tweak the Acceptance Criteria to match the Gherkin steps that are already defined so that you do not have to write new code.
```
Comment (dougnoel) [inline README.md]: A ticket in a ticketing system perhaps?

File: README.md [hunk 7]
```diff
@@ -59,16 +59,16 @@ cp conf/sentinel.linux.example.yml conf/sentinel.yml
 mvn test
 ```
 
-If chrome pops up and the tests run successfully, you are ready to move to Section 1.2.
+If Google Chrome pops up and the tests run successfully, you are ready to move to Section 1.2.
 
 If you have issues with installation, refer to the individual tool manufacturers.
 
-## 1.2 Running the Example Tests on the Command Line
+## 1.2 Running the Example Tests on the Command Line (Terminal for Mac Users)
```
Comment (dougnoel) [inline README.md]: Command Line is actually the generic term. See: https://en.wikipedia.org/wiki/Command-line_interface

File: README.md (hunk 4 above)
Comment (OliviaRudiak) [reply README.md:142]: Will update 4/1
-Olivia

**Pull request merged**